dco-signoff-hook/README.md

```diff
@@ -5,7 +5,7 @@ You can also configure it globally (for every repo on your machine) by copying t
 
 You can also sign off your contributions manually by doing ONE of the following:
 * Use `git commit -s ...` with each commit to add the sign-off or
-* Manually add a `Signed-off-by: Your Name <your.email@example.com>` to each commit message.
+* Manually add a `Signed-off-by: Your Name <your.email@example.com>` to each commit message; please note that Name and Email of sign-off must match the commit's Author, in other words the `user.name` and `user.email` of the git configuration used when creating the commit. Using `git commit -s ...` does this automatically.
```

Comment:
I am not sure if this is too verbose here for an alternative step. We should recommend the automatic way instead of the manual way.

Reply:
thank @terrytangyuan , the automatic way is listed just above this bulletpoint.

this is imho nice "gotchas" for those situation where you have to resolve something manually; it happen a couple of times to me, and in the first occurence I lost time digging for this information, so my hope is this will spare some time for others facing potentially same issues when those rare occurences of having to manually signing off

Reply:
Were there any error messages when the name/email mismatches? I wonder if we should defer this to the DCO check itself since I'd imagine there are many other gotchas. This one talks about the name but there can be a mismatch in primary email as well

Reply:
> This one talks about the name but there can be a mismatch in primary email as well

this comment actually refers to the commit Author, as a whole, making reference to the Name and Email explicitly. 


> I wonder if we should defer this to the DCO check itself since I'd imagine there are many other gotchas.

I've added this comment after some little frustration because I didn't find this information easily, and my hope would have been to spare others of the same.

If we think however this is resulting in the doc instead being too verbose, we can close this PR.

Reply:
Ok if DCO check doesn't provide much information then this would be useful to have. Thanks for clarifying.